specs/altair/validator.md

```diff
@@ -229,7 +229,7 @@ def process_sync_committee_contributions(block: BeaconBlock,
         subcommittee_index = contribution.subcommittee_index
         for index, participated in enumerate(contribution.aggregation_bits):
             if participated:
-                participant_index = SYNC_COMMITTEE_SIZE // SYNC_COMMITTEE_SUBNET_COUNT * subcommittee_index + index
+                participant_index = int(SYNC_COMMITTEE_SIZE // SYNC_COMMITTEE_SUBNET_COUNT * subcommittee_index + index)
```

Comment:
the `remerkleable` definition wants an `int` and the operations w/ the constants pollutes the type to `uint64` which led to some problems when setting values in the underlying `Bitvector` `sync_committee_bits`

Reply:
@protolambda do you have any thoughts or preferences about the typing here -- the indexing method for `remerkleable`'s `Bitvector` wants an `int` but could take any "int"-like type, especially one internal to the `remerkleable` library

Reply:
`uint8, uint16, ` etc. in remerkleable are all sub-types of `int`. Even `boolean` is (just like python `bool` is too). There should be no problem using it as argument in a list-lookup. Maybe the type is lost somewhere? I'm not sure what error/warning you're seeing.

Reply:
rolling back this change and waiting until we can pull this into the specs repo: https://github.com/protolambda/remerkleable/pull/11